src/locales/en.json

```diff
@@ -43,5 +43,6 @@
 	"FETCHED_ROLE_PERMISSION_SUCCESSFULLY": "permissions list fetched successfully",
 	"ORG_EXTENSION_CREATED_SUCCESSFULLY": "Organization Extension Created Successfully",
 	"PERMISSIONS_NOT_FOUND": "permissions not found",
-	"INVALID_DATA_PASSED": "Invalid data passed for creation"
+	"INVALID_DATA_PASSED": "Invalid data passed for creation",
```

Comment:
@adithyadinesh0412 remove all unused variables from the list

Reply:
Almost other variables can come in our use case also, so i've retained it so that we need not do rework to add it again.